Source/WebCore/DerivedSources.make

```diff
@@ -1768,7 +1768,6 @@ MODERN_MEDIA_CONTROLS_STYLE_SHEETS = \
     $(WebCore)/Modules/modern-media-controls/controls/media-controls.css \
     $(WebCore)/Modules/modern-media-controls/controls/media-document.css \
     $(WebCore)/Modules/modern-media-controls/controls/placard.css \
-    $(WebCore)/Modules/modern-media-controls/controls/range-button.css \
```

Comment:
I think you also have to remove these from `Source/WebCore/CMakeLists.txt` and `Source/WebCore/DerivedSources-input.xcfilelist`.